pkg/pkg.bzl

```diff
@@ -403,6 +413,10 @@ def pkg_tar(name, **kwargs):
     pkg_tar_impl(
         name = name,
         out = kwargs.pop("out", None) or (name + "." + extension),
+        private_stamp_detect = select({
+            _stamp_condition: True,
+            "//conditions:default": False,
+        }),
```

Comment:
If this is going to be propagated to other rules, it would be nice if we didn't have to put macros in front of them to provide this feature.

Is there a way to propagate a value of a `config_setting` into a rule without passing it in as an argument in a `select`?  I briefly looked into embedding this into a [`build_setting`](https://docs.bazel.build/versions/master/skylark/config.html#user-defined-build-settings), but the docs made it seem like the `build_setting_default` couldn't itself be configured.  I didn't look very hard, FWIW.